lib/src/scss-shared/_nx-text-helpers.scss

```diff
@@ -90,6 +99,7 @@
 
   // "code" used below to support Markdown text in NxVulnerabilityDetails
   code, .nx-code {
+    @include monospace();
```

Comment:
So you want `.nx-code` to be normal weight even when it's in something that's otherwise bold (like a header)? That's not the effect that this PR had before you added the mixins

Reply:
> So you want `.nx-code` to be normal weight even when it's in something that's otherwise bold (like a header)? That's not the effect that this PR had before you added the mixins

Ah good point. https://github.com/sonatype/sonatype-react-shared-components/pull/273/commits/1b7e411138b982aa98cb77c2652340f48de4ae7a